modules/data-model/links/api/src/main/java/io/uhndata/cards/links/api/LinkUtils.java

```diff
@@ -56,6 +59,25 @@ public interface LinkUtils
      */
     Collection<Link> getLinks(Node source);
 
+    /**
+     * Retrieve all the links from a resource of a desired link type.
+     *
+     * @param source the node to get links from
+     * @param linkName The name of the link type, or a path starting from the root node to the link typs
```

Comment:
To be consistent with the other APIs where we have a link type:

```suggestion
     * @param type the type of link to create, the name of the node holding the link definition, or a JCR path to it
```